commands/listdatabreaches.js

```diff
@@ -0,0 +1,44 @@
+const { SlashCommandBuilder, EmbedBuilder } = require("discord.js");
+const axios = require('axios');
+
+require('dotenv').config()
+
+function getBreaches(breaches) {
+	let array = []
+	breaches.forEach(breach => {
+		exampleEmbed = new EmbedBuilder()
+		.setColor(0x0099FF)
+		let {PwnCount, Title, Domain, BreachDate, AddedDate,Description, LogoPath, ModifiedDate, DataClasses } = breach
+      			exampleEmbed
+				.addFields(
+				{ name: 'Title: ', value: Title, inline: true },
+				{ name: 'Domain: ', value: Domain, inline: true },
+				{ name: 'Pwn Count: ', value: String(PwnCount), inline: true },
+				{ name: 'Description: ', value: Description },
```

Comment:
@Miablo I tested your command and noticed that the api responds with html instead of plan text.
Since I didn't find another solution on how to show html, it might be better to remove the description entirely.
What do you think?
![image](https://user-images.githubusercontent.com/41455100/193854561-53bdd294-4c44-4ef9-acd9-57da77dac027.png)

Reply:
Maybe this would be the solution https://www.npmjs.com/package/node-html-to-image

Reply:
yeah let me look into it and see what I can do but worst case I can remove the description!

Reply:
This is what it looks like using: npm install html-to-text

<img width="580" alt="Screen Shot 2022-10-04 at 8 51 00 PM" src="https://user-images.githubusercontent.com/19197386/193956225-7159288c-c804-476b-81bd-5a044134b98e.png">

Reply:
well done!